scripts/generate_presigned_url.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#Note: DONT run as root
+
+# 1. upload test files (duckdb/data/) to testing Minio server
+# 2. use `usr/bin/mc share download` to generatd the presigned url
+# 3. set presigned url to env variable, so sqllogictest can read the presigned url
+
+# 1
+build/release/test/unittest test/sql/copy/s3/s3_presigned_upload.test
```

Comment:
What's the idea behind these two lines?

Reply:
I explain in the comment below

Reply:
@douenergy Ah that makes! I think it could be a bit confusing though to run these tests for uploading, maybe a nice solution here would be to mount the `./data` folder in the `minio_setup` service and then `mc cp` some test files to the buckets for testing?

Reply:
@samansmink I think that mounts work well for small CSV and Parquet data that are already present in the DuckDB repository. However, I have also tested presigned url with larger Parquet data, such as the lineitem from the TPC-H extension, which I need to leverage DuckDB's great power.

For larger Parquet data, maybe we can use the following command in generate_presigned_url script:
`duckdb -c "CALL DBGEN(sf=1)"` 
`duckdb -c "COPY lineitem TO 's3://test-bucket. ..' (FORMAT 'parquet');"`  

what do you think ?

Reply:
@douenergy sure that sounds good to me!